CHANGELOG.md

```diff
@@ -40,6 +48,27 @@ All notable changes to this project will be documented in this file.
     * #### Added
     * #### Changed
         * `Session.fetch()` and `Session.fetch_into()` now takes additional parameters for associated attributes and add resulting record as part of the waveform info
+        * Enum values and attribute names that start with an underscore + digit have been renamed
+            * `Session._5102_adjust_pretrigger_samples` --> `Session.adjust_pretrigger_samples_5102`
+            * `Session._5v_out_output_terminal` --> `Session.five_v_out_output_terminal`
+            * `ExportableSignals._5V_OUT` --> `ExportableSignals.FIVE_V_OUT`
```

Comment:
Are these based on the corresponding .NET API changes?

Reply:
No.

However,
`NISCOPE_ATTR_5V_OUT_OUTPUT_TERMINAL` ==> `FiveVoltOutputTerminal`
`NISCOPE_ATTR_5102_ADJUST_PRETRIGGER_SAMPLES` ==> not found
`NISCOPE_VAL_5V_OUT` ==> `FiveVoltOutputTerminal` (usage doesn't match up though)